libr/anal/p/anal_x86_cs.c

```diff
@@ -1403,6 +1403,42 @@ static void anop_esil (RAnal *a, RAnalOp *op, ut64 addr, const ut8 *buf, int len
 	}
 }
 
+static int parse_reg_name_mov(RRegItem *reg, csh *handle, cs_insn *insn, int reg_num) {
+	if (!reg) { return -1; }
+
+	switch (INSOP(reg_num).type) {
+	case X86_OP_REG:
+		reg->name = cs_reg_name (*handle, INSOP(reg_num).reg);
+		break;
+	default:
+		break;
+	}
+
+	return 0;
+}
+
+static int parse_reg_name_lea(RRegItem *reg, csh *handle, cs_insn *insn, int reg_num) {
+	if (!reg) { return -1; }
```

Comment:
i never told you to do that
> On 04 Sep 2016, at 14:53, Paul <notifications@github.com> wrote:
> 
> In libr/anal/p/anal_x86_cs.c <https://github.com/radare/radare2/pull/5660#discussion_r77451931>:
> 
> > +static int parse_reg_name_mov(RRegItem *reg, csh *handle, cs_insn *insn, int reg_num) {
> > +	if (!reg) { return -1; }
> > +
> > +	switch (INSOP(reg_num).type) {
> > +	case X86_OP_REG:
> > +		reg->name = cs_reg_name (*handle, INSOP(reg_num).reg);
> > +		break;
> > +	default:
> > +		break;
> > +	}
> > +
> > +	return 0;
> > +}
> > +
> > +static int parse_reg_name_lea(RRegItem *reg, csh *handle, cs_insn *insn, int reg_num) {
> > +	if (!reg) { return -1; }
> To be honest it makes me mad. If i write
> 
> if (something) {
>     ;
> }
> You ask me to change it to
> 
> if (something) {}
> And if I write in one line, you ask to change it back :/
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub <https://github.com/radare/radare2/pull/5660/files/0aed0b17ecd8c838f6671a9ef45da67b3638fc04#r77451931>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AA3-lpv_ckFNlIHStsw5EogEybA1n-Jhks5qmr9fgaJpZM4JxcWx>.
>